datumaro/plugins/voc_format/converter.py

```diff
@@ -367,8 +367,10 @@ def save_action_lists(self, subset_name, action_list):
         items = {k: True for k in action_list}
         if self._patch and osp.isfile(ann_file):
             self._get_filtered_lines(ann_file, self._patch, subset_name, items)
-        with open(ann_file, 'w') as f:
+        with open(ann_file, 'w', encoding='utf-8') as f:
             for item in items:
+                if 1 < len(item.split()):
```

Comment:
I don't think this solution is right. All the VOC subformats should handle such names without changes. The only problematic subformat should be `Layout`, where only parsing should be changed.